### ISSUE TITLE
sonar-runner dependency isn't found on Gradle 3.x

### ISSUE THREAD

**Issue by michaelklishin** [rabbitmq/hop#85]

**Comment by michaelklishin** [comment]
@artembilan any chance you have seen this before?

**Comment by garyrussell** [comment]
@michaelklishin The new task is `sonarqube` - looks like we switched when we went from 2.5 to 2.6...

https://github.com/spring-projects/spring-amqp/commit/2594d01786629f21d240c6ea9f4c13cdd6e0eb2a

**Comment by michaelklishin** [comment]
@garyrussell may I ask you to submit a PR that switches Hop to it (and Gradle 3)? Pretty please :)

**Comment by artembilan** [comment]
@michaelklishin , yes, @garyrussell gives you correct answer.
That must be like this now:
```
plugins {
     id "org.sonarqube" version "1.2"
}

sonarqube {
	properties {
    property "sonar.jacoco.reportPath", "${buildDir.name}/jacoco.exec"
    property "sonar.links.homepage", linkHomepage
    property "sonar.links.ci", linkCi
    property "sonar.links.issue", linkIssue
    property "sonar.links.scm", linkScmUrl
    property "sonar.links.scm_dev", linkScmDevConnection
    property "sonar.java.coveragePlugin", "jacoco"
  }
}
```
But bare in mind: there is some incompatibility between Gradle 3.x and our Bamboo. The last one is not able to find test reports somehow.
So, the latest possible Gradle version right now is `2.14.1`

**Comment by michaelklishin** [comment]
@artembilan so you're saying that we should not move to Gradle 3 just yet and thus this issue should be tackled when we can/choose to do so?

**Comment by garyrussell** [comment]
@artembilan - if gradle 3 puts the test results in a different place, can you not change the plan to point to that location? Or is it a bigger issue?

<img width="540" alt="screen shot 2016-10-06 at 9 12 14 am" src="https://cloud.githubusercontent.com/assets/483832/19153583/2613324e-8ba5-11e6-97b8-800c521c518b.png">

**Comment by artembilan** [comment]
Something like that.
I poke upgrades from time to time and yes in between `2.5` and `2.6` upgrades it requested from us to use `id "org.sonarqube" version "1.2"`.
Since there we have just move up to `2.14`.
The Gradle `3.x` wasn't successful from the integration perspective. One more issue from our side is IO Platform which doesn't support Gradle `3.x` facet yet.

@garyrussell , I tried that, indeed: https://docs.gradle.org/3.0/release-notes#unique-default-test-result-and-report-directories-for-test-tasks. Or I did something wrong there, or just have missed something to fix else. I can ask Rob Winch if you really are interested in Gradle `3.x` upgrade. At least I see that SS is already there: https://github.com/spring-projects/spring-security/commit/22fe8b33c6d0e1d40dec421b95b5935720cb4889 ...
Hm `Aug 30`... I'm really missing something... :cry:
